app/controllers/kuroko2/job_definitions_controller.rb

```diff
@@ -2,7 +2,7 @@ class Kuroko2::JobDefinitionsController < Kuroko2::ApplicationController
   before_action :set_definition, only: [:show, :edit, :update, :destroy]
 
   def index
-    rel = Kuroko2::JobDefinition
+    rel = Kuroko2::JobDefinition.joins(:admins).includes(:tags, :job_instances, :job_schedules, :admins)
```

Comment:
why do you join admins not only including it? (I don't know about table structures)